### DISCUSSION TITLE
What does disableOptimizedLoading do?

### DISCUSSION THREAD

**Discussion by jojov92** [vercel/next.js#44829] [Help] [answered]
### Summary

Hello 🙂 

I had a few questions around disableOptimizedLoading. We initially had it set to true when we were using Next11 and now we are using Next12 but in the next12 files(https://github.com/vercel/next.js/blob/v12.2.5/packages/next/server/config-shared.ts#L557),  disableOptimizedLoading is set to false. 

I am not sure if we should set it to true or false now?

We have a custom next js configuration.

Thank you very much!

**Comment by amyegan** [comment, marked as answer]
Hi @jojov92! I checked with the Next.js team, and they said you should not set that config option manually. It is an experimental setting that could be changed or removed at any time and is meant for internal testing.

**Reply by jojov92** [reply]
ah so we should set it back to `false` then?

**Reply by amyegan** [reply]
You shouldn't need to set it at all, so you can remove it from your config to allow the default value to be used 🙂

**Reply by jojov92** [reply]
ok thank you very much :)

**Reply by jojov92** [reply]
@amyegan would you recommend us using the `next script` instead to deal with script optimisation and preloading?

**Reply by amyegan** [reply]
Yes, that is a good solution
